src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -863,3 +877,41 @@ function getQuestionIdSuffix(questionNum) {
     return idSuffix;
 }
 
+function toggleCustomFeedbackPathsDisplay(toggleLink) {
```

Comment:
I think I would prefer to do it at a later stage, at least after the PR has been merged in (I will be integrating master back into the `custom-feedback-paths` branch periodically), so that it resolving conflicts will be (hopefully) less messy.